src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -1224,6 +1225,12 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
       ...this.mailData.cc.map((cc: any) => cc.email.toLowerCase()),
       ...this.mailData.bcc.map((bcc: any) => bcc.email.toLowerCase()),
     ];
+    // Subject Waning Message
+    if (this.mailData.subject.length === 0) {
+      this.store.dispatch(new SnackErrorPush({ message: 'Subject Line is empty.' }));
```

Comment:
We have to add this new string to the translation files.
Add it in English to all the languages so we can forward them to translators often.